src/kbmod_wf/multi_night_workflow.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import os
+
+import toml
+import parsl
+from parsl import python_app, File
+import parsl.executors
+
+from kbmod_wf.utilities import (
+    apply_runtime_updates,
+    get_resource_config,
+    get_executors,
+    get_configured_logger,
+)
+
+from kbmod_wf.workflow_tasks import create_manifest, ic_to_wu, reproject_wu_multi_night, kbmod_search
```

Comment:
Is `reproject_wu_multi_night` still required as an import here? I might be missing something, but it seems like things are taken care of in the reproject function defined on line 24.

Reply:
Yes you're correct. That was left over from an earlier version, so cleaned that up.